test/gie/builtins.gie

```diff
@@ -1162,6 +1162,112 @@ accept  -2 -1
+accept 72 46
+expect 9643052.686797861 23470487.369125906
+accept 145 75
+expect 7737390.782312192 20104788.054769896
+accept -77 61
+expect 9668545.797051517 16783947.67041814
+accept -26 35
+expect 15441270.037205558 20068684.009483118
+accept 29 -13
+expect 15454500.775444025 26772291.248060163
+accept 72 -25
+expect 9646854.070149897 30174627.37899145
+accept 96 10
+expect 7748510.227719365 26793197.86531571
+accept 169 35
+expect 3879364.926075886 20112884.602338977
+accept -151 13
+expect 3855457.9788894877 13372404.237126907
+accept -108 25
+expect 7757309.887436695 13427462.802436445
+accept -84 -10
+expect 9655653.729867224 10046033.288760712
+accept -42 -5
+expect 13531956.757129436 10009950.782333512
+accept -11 -35
+expect 13524799.031510707 3365720.025783973
+accept 154 -35
+expect 1943420.8175922546 30184052.521318957
+accept -157 -28
+expect 1942090.2539895012 10076591.20216224
+accept -108 -46
+expect 7761111.270788729 6723322.792570909
+accept -35 -75
+expect 9666773.175274398 3357623.4782149
+accept 98 -48
+expect 4871988.820643149 32910398.56714167
+accept 113 -72
+expect 7725850.483813245 1088496.8256700835
+accept 143 -9
+expect 3215425.0474725077 27916795.439275663
+accept 124 7
+expect 5140625.112062726 26778848.513305675
+accept 147 15
+expect 2546085.5873131985 22379298.380909223
+
+direction inverse
+accept 13500000 23400000
+expect 23.028334260781396 29.00141845366607
+accept 9700000 23400000
+expect 70.91748976033631 46.69938767448572
+accept 7700000 20100000
+expect 145.97790022956485 74.66181692499651
+accept 9700000 16700000
+expect -76.80102747900565 59.99374652248361
+accept 15500000 20100000
+expect -25.944389921688217 34.24636404674132
+accept 15500000 26800000
+expect 28.780505935021033 -13.567140588707066
+accept 9700000 30100000
+expect 71.4679674085186 -24.079311453738754
+accept 7700000 26800000
+expect 96.5497950965985 9.88121445991218
+accept 3900000 20100000
+expect 169.13009208849755 35.254403049754224
+accept 3900000 13400000
+expect -150.64046965868116 13.481494624013168
+accept 7700000 13400000
+expect -108.66196431422468 24.604190039126202
+accept 9700000 10000000
+expect -83.45176986114532 -10.484774451578211
+accept 13500000 10000000
+expect -42.33857403889376 -5.173528026525927
+accept 13500000 3300000
+expect -10.156579309260813 -35.39429168339308
+accept 1900000 30100000
+expect 154.6981118584859 -34.09548216619245
+accept 1900000 10000000
+expect -157.1486568493204 -28.983938091775162
+accept 7700000 6700000
+expect -109.0368655953666 -46.16466623109972
+accept 9700000 3300000
+expect -32.11127170743792 -74.98612150074251
+accept 4800000 32900000
+expect 98.90848938837188 -48.50784207235999
+accept 7700000 1100000
+expect 114.00231130949376 -71.99425265168686
+accept 3200000 27900000
+expect 143.20066829467564 -8.847739058325072
+accept 5200000 26800000
+expect 123.33794956763194 6.861286881197438
+accept 2600000 22300000
+expect 147.63769509437498 15.818879769693432
```

Comment:
@plouvart could you add a test point that triggers the out-of-projection-domain error ?

Reply:
Yes. I'll do that in a few minutes